plugins/module_utils/redfish_utils.py

```diff
@@ -3051,3 +3052,62 @@ def get_manager_inventory(self, manager_uri):
 
     def get_multi_manager_inventory(self):
         return self.aggregate_managers(self.get_manager_inventory)
+
+    def set_sessionservice(self, sessionservice_config):
```

Comment:
Maybe a bit of nit picking but, wouldn't it be better off as:
```suggestion
    def set_session_service(self, session_service_config):
```

Reply:
Thanks for the feedback! I modeled this based on the existing code. I will make the changes asap.